ewah/operators/s3.py

```diff
@@ -132,8 +143,8 @@ def read_key(key_name):
         all_objects = [
             o
             for o in page_iterator.build_full_result()["Contents"]
-            if (not modified_from or o["LastModified"] >= modified_from)
-            and (not modified_until or o["LastModified"] < modified_until)
+            if (not modified_from or o["LastModified"] > modified_from)
+            and (not modified_until or o["LastModified"] <= modified_until)
```

Comment:
Question (non-blocking): Why switch this?

Reply:
I was causing duplicates when running the incremental load multiple times.

In summary we were including files that had greater or _equal_ last modified date than  max(last_modified_date). This was causing the file with same last modified date to be loaded again

Reply:
Of course it did. Makes perfect sense. Thanks a lot!!